packages/angular/package.json

```diff
@@ -43,10 +44,12 @@
     "tslib": "^2.3.1"
   },
   "peerDependencies": {
+    "@unovis/shared": "^1.2.1",
```

Comment:
Since  `@unovis/shared` is not a public package I would change it to ` "@unovis/shared": "*"`. It'll make library versioning easier, otherwise we'll have to update our versioning scripts.